docs/static/releasenotes.asciidoc

```diff
@@ -76,12 +76,39 @@ This section summarizes the changes in the following releases:
 === Logstash 8.17.0 Release Notes
 
 [[known-issues-8-17-0]]
-==== Known issue
+==== Known issues
+
+[[known-issue-8-17-0-jvm]]
+===== JVM version changes needed when upgrading {ls} from 8.12.0 (or earlier)
+
+If the `jvm.options` file was modified and not overwritten with the newest version, you may see a "deserialize invocation error" message, causing the pipeline to crash. 
+Users are affected if the Persistent Queue (PQ) is enabled, and the pipeline is processing messages larger than 20MB.
+
+**Solution:** Apply the default change contained in the newer 'jvm.options' file, as seen in this https://github.com/elastic/logstash/blob/main/config/jvm.options#L66-L82[example].
+
+[[known-issue-8-17-0-json_lines]]
+===== "Input buffer full" error with {ls} 8.16.0, 8.16.1, or 8.17.0
+
+If you are using `json_lines` codec 3.2.0 (or later) with {ls} 8.16.0, 8.16.1, or 8.17.0, you may see an error similar to this one, crashing the pipelines:
+```
+unable to process event. {:message=>"input buffer full", :class=>"Java::JavaLang::IllegalStateException", :backtrace=>["org.logstash.common.BufferedTokenizerExt.extract(BufferedTokenizerExt.java:83)", "usr.share.logstash.vendor.bundle.jruby.$3_dot_1_dot_0.gems.logstash_minus_codec_minus_json_lines_minus_3_dot_2_dot_2.lib.logstash.codecs.json_lines.RUBY$method$decode$0(/usr/share/logstash/vendor/bundle/jruby/3.1.0/gems/logstash-codec-json_lines-3.2.2/lib/logstash/codecs/json_lines.rb:69)", "org.jruby.internal.runtime.methods.CompiledIRMethod.call(CompiledIRMethod.java:165)", "org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:185)",
+```
+The issue was fixed in https://github.com/elastic/logstash/pull/16760.
+
+This problem is most likely to be seen when you are using the <<plugins-integrations-logstash,{ls} integration>> plugin to ship data between two {ls} instances, but may appear in other situations, too. 
+
+**Workaround for {ls}-to-{ls} communication** 
 
-When upgrading Logstash to 8.12.0 or earlier, if the `jvm.options` was previously modified and not overwritten with the newest version,
-users may see a "deserialize invocation error" message causing the pipeline to crash. Users are affected if the Persistent Queue
-is enabled and the pipeline is processing messages larger than 20MB.
-The solution is to apply the default change contained in the newer 'jvm.options' file, as seen https://github.com/elastic/logstash/blob/main/config/jvm.options#L66-L82[here].
+The {ls}-to-{ls} issue can be mitigated by:
+
+* Downgrading the _receiving_ {ls} to `8.16.2`, or any {ls} in the `8.15` series,  **_AND/OR_**
+* Upgrading the {ls} integration filter of the _sending_ {ls} to version `1.0.4`.
```

Comment:
```suggestion
* Upgrading the <<plugins-integrations-logstash,{ls} integration>> plugin of the _sending_ {ls} to version `1.0.4`.
```

This should be the Logstash integration plugin, right?  The agent_integration filter doesn't make sense to me.  

Note to self @karenzone:  Make sure any change here gets propogated to 8.16.1 and 8.16.0 sections.

Reply:
Correct - it's the logstash input and output that is affected.